### PR TITLE
Review cleanup for PR #2344.

### PR DESCRIPTION
**Pull request by dhermes**
@daspecster After this is in, feel free to run with the implementation (e.g. system tests, all features, etc.)

Remaining review issues not fixed by this PR:

- https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2344/files#r79738759
- https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2344/files#r79307666

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -237,4 +237,4 @@ Cloud Storage
   client = storage.Client()
   bucket = client.get_bucket('<your-bucket-name>')
   blob = bucket.blob('my-test-file.txt')
-  blob.upload_from_string('this is test content!')
\ No newline at end of file
+  blob.upload_from_string('this is test content!')
```
Comment (daspecster) [inline docs/index.rst]: Did you figure out what this is? I'm guessing line endings?

File: docs/speech-usage.rst [hunk 2]
```diff
@@ -38,17 +38,18 @@ and returns alternative text transcriptons.
...
-     ...                 "FLAC", 16000, max_alternatives=2):
+     >>> alternatives = client.sync_recognize(
+     ...     None, 'gs://my-bucket/recording.flac',
+     ...     'FLAC', 16000, max_alternatives=2)
      >>> for alternative in alternatives:
      ...     print('=' * 20)
-     ...     print('   transcript: %s' % (alternative["transcript"],))
-     ...     print('   confidence: %s' % (alternative["confidence"],))
+     ...     print('transcript: ' + alternative['transcript'])
+     ...     print('confidence: ' + alternative['confidence'])
```
Comment (daspecster) [inline docs/speech-usage.rst]: Why is this preferred? Easier to read?

File: docs/index.rst (hunk 1 above)
Comment (dhermes) [inline docs/index.rst]: Yeah there was a missing one. I went through and made sure all whitespace issues were resolved.

File: docs/speech-usage.rst (hunk 2 above)
Comment (dhermes) [inline docs/speech-usage.rst]: Yeah I was changing the double to single quotes and then figured I'd make it shorter too. Singleton tuples are no fun.

**Comment by tseaver** [comment]
LGTM

**Comment by daspecster** [comment]
Did travis slow down on us again today?

**Comment by tseaver** [comment]
@daspecster We can apparently get throttled due to builds throughout the organization.

**Pull request merged**